new_tab_custom_iterm_script_iterm_3.1.1.applescript

```diff
@@ -0,0 +1,47 @@
+-- This is v0.7 of the custom script for AlfredApp for iTerm 3.1.1+
+-- created by WangJia hmfight@qq.com
+-- github: https://github.com/hmfight
+
+on alfred_script(q)
+	if application "iTerm2" is running or application "iTerm" is running then
+		run script "
+			on run {q}
+				tell application \":Applications:iTerm.app\"
```

Comment:
It’s safer to use `tell application \"iTerm\"`. You can’t guarantee the user will have iTerm2 in `/Applications`.